src/TortoiseProc/LogDlg.cpp

```diff
@@ -431,10 +455,35 @@ BOOL CLogDlg::OnInitDialog()
 	reg.Replace(_T(':'),_T('_'));
 	m_History.Load(reg, _T("ref"));
 
+	m_FromDateHistory.SetMaxHistoryItems((LONG)CRegDWORD(_T("Software\\TortoiseGit\\MaxFromDateHistoryItems"), 5));
+	reg.Format(_T("Software\\TortoiseGit\\History\\LogDlg_FromDates\\%s"), g_Git.m_CurrentDir);
+	reg.Replace(_T(':'), _T('_'));
+	m_FromDateHistory.Load(reg, _T("FromDate"));
+
 	ShowStartRef();
 	return FALSE;
 }
 
+void CLogDlg::OnDestroy()
+{
+	__super::OnDestroy();
+
+	m_Brush.DeleteObject();

```

Comment:
1. Old habits die hard!
Create in OnInitDialog(), destory in OnDestory()

2. I am not sure if it is suitable to delete GDI object in destructor.
Is it OK?